src/gyp.py

```diff
@@ -0,0 +1,8 @@
+# Dummy gyp.py because chromium/src/vs_toolchain.py depends on gyp, but it's in
```

Comment:
Never mind; I decided to go ahead and use v8's script, to ensure that the same compiler version is used for v8 and the other builds (turns out v8 isn't quite in sync with Chromium).